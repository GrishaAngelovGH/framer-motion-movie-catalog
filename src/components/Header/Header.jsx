import { motion } from "framer-motion"
import { Link } from "react-router-dom"

import filmRoll from "/images/film-roll.png"

const Header = () => (
  <div className="flex justify-between items-center bg-blue-500 h-full p-2">
    <div className="ms-5 flex items-center">
      <img src={filmRoll} className="w-[40px]" />
      <h1 className="ms-3 text-2xl font-bold text-blue-900">Cinema</h1>
    </div>

    <div className="me-5">
      <Link to="/" className="text-white hover:text-white hover:underline me-5">Home</Link>
      <Link to="/favorites" className="text-white hover:text-white hover:underline me-5">Favorites</Link>
    </div>
  </div>
)

export default Header