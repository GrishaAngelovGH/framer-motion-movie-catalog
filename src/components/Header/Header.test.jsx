import { render } from "@testing-library/react"
import { BrowserRouter as Router, Routes, Route } from "react-router-dom"

import Header from "./Header"

test("should render Header component", () => {
  const view = render(
    <Router>
      <Routes>
        <Route path="/" element={<Header />} />
      </Routes>
    </Router>
  )

  expect(view).toMatchSnapshot()
})