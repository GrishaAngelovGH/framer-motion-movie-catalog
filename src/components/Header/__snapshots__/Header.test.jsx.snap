// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render Header component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex justify-between items-center bg-blue-500 h-full p-2"
      >
        <div
          class="ms-5 flex items-center"
        >
          <img
            class="w-[40px]"
            src="/images/film-roll.png"
          />
          <h1
            class="ms-3 text-2xl font-bold text-blue-900"
          >
            Cinema
          </h1>
        </div>
        <div
          class="me-5"
        >
          <a
            class="text-white hover:text-white hover:underline me-5"
            href="/"
          >
            Home
          </a>
          <a
            class="text-white hover:text-white hover:underline me-5"
            href="/favorites"
          >
            Favorites
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex justify-between items-center bg-blue-500 h-full p-2"
    >
      <div
        class="ms-5 flex items-center"
      >
        <img
          class="w-[40px]"
          src="/images/film-roll.png"
        />
        <h1
          class="ms-3 text-2xl font-bold text-blue-900"
        >
          Cinema
        </h1>
      </div>
      <div
        class="me-5"
      >
        <a
          class="text-white hover:text-white hover:underline me-5"
          href="/"
        >
          Home
        </a>
        <a
          class="text-white hover:text-white hover:underline me-5"
          href="/favorites"
        >
          Favorites
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
